Stonith/docs/3nodes/scsi_stonith.md

```diff
@@ -0,0 +1,683 @@
+### Setup on 3 VM's
+
```

Comment:
Can you separate scsi and ipmi
and add in https://github.com/Seagate/cortx-experiments/tree/main/Stonith/docs
depend on scsi and ipmi